src/Microsoft.VisualStudio.AppDesigner/PublicAPI.Shipped.txt

```diff
@@ -351,6 +351,22 @@ Microsoft.VisualStudio.Editors.Constants.New() -> Void
 Microsoft.VisualStudio.Editors.IVBPackage
 Microsoft.VisualStudio.Editors.IVBPackage.GetLastShownApplicationDesignerTab(projectHierarchy As Microsoft.VisualStudio.Shell.Interop.IVsHierarchy) -> Integer
 Microsoft.VisualStudio.Editors.IVBPackage.SetLastShownApplicationDesignerTab(projectHierarchy As Microsoft.VisualStudio.Shell.Interop.IVsHierarchy, tab As Integer) -> Void
+Microsoft.VisualStudio.Editors.MyApplication.IVsMyApplicationProperties.AuthenticationMode() -> Integer
```

Comment:
This was a bug fix in the public API analyzer.

Reply:
The old "forget VB exists" bug? :)